data/json/monsters/defense_bot.json

```diff
@@ -140,6 +140,38 @@
       "STUN_IMMUNE"
     ]
   },
+  {
+    "id": "mon_secubot_flesh",
+    "type": "MONSTER",
+    "copy-from": "mon_secubot",
+    "name": { "str": "writhing puppet TALON UGV" },
+    "description": "This was once a TALON unmanned ground vehicle equipped with an M16A4.  However, it has been completely engulfed in missmatched flesh.  As its mechanical bits attempt to move, blood oozes from its sinewy chassis.  Though it's treads are non-functional a mass of fingers seems to slowly drag its hulking frame.",
```

Comment:
Couple more, sorry!
`missmatched` -> `mismatched`
`Though it's treads ` -> `Though its treads`
`non-functional a` -> `non-functional, a`